src/app/components/Search/Search.jsx

```diff
@@ -134,46 +137,55 @@ class Search extends React.Component {
         method="POST"
         className="nypl-omnisearch-form"
       >
-        <div className={`nypl-omnisearch nypl-text-field ${inputError ? 'nypl-field-error' : ''}`}>
-          <span className="nypl-omni-fields">
-            <label htmlFor="search-by-field">Search in</label>
-            <select
-              id="search-by-field"
-              onChange={this.onFieldChange}
-              value={this.state.field}
-              name="search_scope"
+        <div className="nypl-omnisearch">
+          <div className={`nypl-text-field ${inputError ? 'nypl-field-error' : ''}`}>
+            <span className="nypl-omni-fields">
+              <label htmlFor="search-by-field">Search in</label>
+              <select
+                id="search-by-field"
+                onChange={this.onFieldChange}
+                value={this.state.field}
+                name="search_scope"
+              >
+                <option value="all">All fields</option>
+                <option value="title">Title</option>
+                <option value="contributor">Author/Contributor</option>
+              </select>
+            </span>
+          </div>
+          <div className="nypl-text-field">
+            <span className="nypl-omni-fields">
+              <label htmlFor="search-query" id="search-input-label" className="visuallyhidden">
+                Search for
+              </label>
+              <input
+                type="text"
+                id="search-query"
+                aria-labelledby="search-input-label search-input-status"
+                aria-required="true"
+                placeholder="Keyword, title, or author/contributor"
+                onChange={this.inputChange}
+                value={this.state.searchKeywords}
+                name="q"
+                ref="keywords"
+              />
+            </span>
+          </div>
+          <button className="nypl-omnisearch-button nypl-primary-button">
+            <SearchButton onClick={this.submitSearchRequest} />
```

Comment:
Ah, so `SearchButton` renders a `<button>` element itself. Doing this will result in nested `<button>`s. But I think this was because of the placement of the icon? If that's the case, you can
1) edit the `SearchButton` component and add the `SearchIcon` component there,
2) update this by moving the `onClick` attribute to the top `button` and removing the `SearchButton` component.